torch/_dynamo/variables/builder.py

```diff
@@ -857,6 +857,48 @@ def wrap_listlike(self, value: Union[tuple, list, odict_values, NamedTuple]):
             for i, item in enumerate(value)
         ]
 
+        use_boxed_call = False
```

Comment:
looking for a better way to determine this

Reply:
Jow does this relate to the way aot autograd does it?  cc @Chillee @zou3519

Reply:
AOTAutograd's boxed calling is via [make_boxed_func](https://github.com/pytorch/pytorch/blob/aaef246c74b964ba43f051a4a3e484d25d418f44/torch/_functorch/_aot_autograd/utils.py#L87). The util is used across AOTAutograd to wrap every runtime callable's arguments in a list, forward a [list of inputs created within AOTAutograd](https://github.com/pytorch/pytorch/blob/aaef246c74b964ba43f051a4a3e484d25d418f44/torch/_functorch/aot_autograd.py#L917).

However, because nn.Module and autograd.Function both rely on *args, it's not easy to change the [runtime function signature](https://github.com/pytorch/pytorch/blob/aaef246c74b964ba43f051a4a3e484d25d418f44/torch/_functorch/aot_autograd.py#L913) expecting 1 positional argument per graph arg. So we need the use_boxed_call logic here to create a single graph arg for all compiled autograd lifted tensors.

Reply:
cc @bdhirsh